Pipfile

```diff
@@ -9,13 +9,19 @@ pytest = "*"
 pylint = "*"
 
 [packages]
+fastapi = "*"
 flask = "*"
 python-dotenv = "*"
 requests = "*"
 gunicorn = "*"
```

Comment:
Can be removed now that we're using uvicorn.

Reply:
Actually we are running `uvicorn` workers via `gunicorn`.
https://www.uvicorn.org/deployment/

Reply:
https://fastapi.tiangolo.com/deployment/#alternatively-deploy-fastapi-without-docker